internal/contributebot/process_test.go

```diff
@@ -22,9 +22,15 @@ import (
 )
 
 func TestProcessIssueEvent(t *testing.T) {
+	const (
+		defaultTitle = "foo: bar"
+	)
+
 	tests := []struct {
 		description string
 		action      string
+		title           string // defaults to defaultTitle
```

Comment:
Just to remove branching, use `defaultTitle` explicitly in the test cases.

(Is this gofmt'd?)

Reply:
Done, Done.